app/Platform/Enums/GameSetType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Enums;
+
+enum GameSetType: string
+{
+    case GAME = "game";
```

Comment:
I feel like this needs to be more descriptive. Maybe `GAME_ALTERNATIVES` or `SIMILAR_GAMES`. I could see there being a second category for `GAME_SUBSETS`, but that data may not end up in this table.

Reply:
Adjusted in latest to `SIMILAR_GAMES`.